app/src/main/java/io/github/polysmee/appointments/fragments/AppointmentCreationAddUserFragment.java

```diff
@@ -73,11 +81,24 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
      * store all objects on the activity (buttons, textViews...) in variables
      */
     private void attributeSetters(View rootView) {
+        users = new ArrayList<>();
+        User.getAllUsersIdsAndThenOnce(this::UsersNamesGetter);
         searchInvite = rootView.findViewById(R.id.appointmentSettingsSearchAdd);
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(getActivity(),
+                android.R.layout.simple_dropdown_item_1line, users);
+        searchInvite.setAdapter(adapter);
         btnInvite = rootView.findViewById(R.id.appointmentSettingsBtnAdd);
         invitesList = rootView.findViewById(R.id.appointmentCreationAddsList);
         invites = new HashSet<>();
         removedInvites = new HashSet<>();
+        builder = new AlertDialog.Builder(getActivity());
+    }
+
+    private void UsersNamesGetter(Set<String> allIds) {
+        for(String userId : allIds){
+            User user = new DatabaseUser(userId);
+            user.getNameAndThen((name) -> users.add(name));
```

Comment:
If a user change his name during the lifecycle of the fragment their will still be the old name, but since the lifecycle is very short and the wanted user changes name only rarely it shouldn't be a problem. Just wanted to let you know. Maybe to not forget this you could write a comment justifiying your choice. Simplicity more important here than this small possible error case